pom.xml

```diff
@@ -48,7 +48,7 @@
   </developers>
 
   <properties>
-    <mavenVersion>3.3.9</mavenVersion>
+    <mavenVersion>3.6.0</mavenVersion>
```

Comment:
Are we wanting to update the version here? I thought you were wanting to resolve 3.3.9 support - `master` currently sets this as 3.3.9.....

Reply:
the api is backward compatible, so you can build against 3.6.0 jars and use in 3.3.9. However, I think it is more sensible to keep the 3.3.9 dependencies. I will revert this version update